core/network/src/main/kotlin/JvmUnitTestDemoAssetManager.kt

```diff
@@ -38,4 +39,5 @@ internal object JvmUnitTestDemoAssetManager : DemoAssetManager {
     private val assets = File(properties["android_merged_assets"].toString())
 
     override fun open(fileName: String): InputStream = File(assets, fileName).inputStream()
+    override fun readText(fileName: String): String = File(assets, fileName).readText(StandardCharsets.UTF_8)
```

Comment:
We don't need to expose an additional api in the AssetManager. Simply use the already existing `open` function, and call either `decodeFromStream` and `decodeFromString(input.bufferedReader().readText())`

Reply:
Thanks I'll remove exposed readText.